packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -71,18 +75,18 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 	 */
 	private readonly enabled: boolean = false;
 
-	private readonly sessionID?: string;
-	private readonly continuityID?: string;
+	private sessionID?: string;
```

Comment:
Can we add some simple docs to the updated properties here? Noting why we expect them to be mutated would be helpful.

Reply:
@WayneFerrao I think it'd be better to have the additional note in the docs for each property, maybe make them TSDoc blocks (`/* */`). Also, worth calling out "when the user opts in/out _of reporting telemetry_". As written it might not make much sense to someone that doesn't have the context.